kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1818,6 +1816,49 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.4</version>
```

Comment:
The version is already defined in `kie-parent-metadata` (see https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/pom.xml#L368), so should be reused.